src/plugin.tsx

```diff
@@ -7,7 +7,9 @@ import type {PluginConfig, MuxInputProps, VideoAssetDocument} from './util/types
 export function muxVideoCustomRendering(config: PluginConfig) {
   return {
     components: {
-      input: (props: MuxInputProps) => <Input config={config} {...props} />,
+      input: (props: MuxInputProps) => (
+        <Input config={{...config, ...props.schemaType.options}} {...props} />
```

Comment:
Great work on this, I didn't even consider we weren't taking per-schema configuration into account ✨